driver-core/src/main/com/mongodb/client/model/search/SearchConstructibleBsonElement.java

```diff
@@ -68,10 +72,16 @@ public SearchConstructibleBsonElement defaultBucket(final String name) {
     }
 
     @Override
-    public SearchConstructibleBsonElement fuzzy(final SearchFuzzy option) {
+    public SearchConstructibleBsonElement fuzzy(@Nullable final FuzzySearchOptions... options) {
```

Comment:
Should this have been rolled back - or should it be `fuzzy(FuzzySearchOptions, FuzzySearchOptions...)` ?

Reply:
Sorry, I missed that one. Rolled back and searched through the rest of the code to make sure there are no other changes related to expressing optional parameters as varargs.